data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_feats.lst

```diff
@@ -289,7 +289,7 @@ Wave Strike							CATEGORY:FEAT		TYPE:Combat
-Crane Style							CATEGORY:FEAT		TYPE:Combat.Style					PREABILITY:2,CATEGORY=FEAT,Dodge,Improved Unarmed Strike																																																					PREMULT:1,[PRETOTALAB:2],[PREVARGTEQ:MonkFeatQualify,1]																																																																																																																																																												DESC:Your unarmed fighting techniques blend poise with graceful defense.																																																																																																																																																																								SOURCEPAGE:p.93															BENEFIT:You take only a -2 penalty on attack rolls for fighting defensively. While using this style and fighting defensively or using the total defense action, you gain an additional +1 dodge bonus to your Armor Class.																																																																																																																																																																																																																																									TEMPBONUS:PC|COMBAT|AC|1|TYPE=Dodge		TEMPBONUS:VAR|FightingDefensivelyAttackPenalty|-2
+Crane Style							CATEGORY:FEAT		TYPE:Combat.Style					PREABILITY:2,CATEGORY=FEAT,Dodge,Improved Unarmed Strike																																																					PREMULT:1,[PRETOTALAB:2],[PREVARGTEQ:MonkFeatQualify,1]																																																																																																																																																												DESC:Your unarmed fighting techniques blend poise with graceful defense.																																																																																																																																																																								SOURCEPAGE:p.93															BENEFIT:You take only a -2 penalty on attack rolls for fighting defensively. While using this style and fighting defensively or using the total defense action, you gain an additional +1 dodge bonus to your Armor Class.																																																																																																																																																																																																																																									TEMPBONUS:PC|COMBAT|AC|1|TYPE=Dodge		BONUS:VAR|FightingDefensivelyAttackPenalty|-2
```

Comment:
Fix the syntax please TEMPBONUS:PC|VAR|xxx

Reply:
Done